src/Microsoft.CodeQuality.Analyzers.Exp/UnitTests/Design/ValidateArgumentsOfPublicMethodsTests.cs

```diff
@@ -1543,12 +1574,15 @@ public void M2(string str, C c)
     }
 }
 ",
-            // Test0.cs(19,17): warning CA1062: In externally visible method 'void Test.M1(string str, C c)', validate parameter 'c' is non-null before using it. If appropriate, throw an ArgumentNullException when the argument is null or add a Code Contract precondition asserting non-null argument.
-            GetCSharpResultAt(19, 17, "void Test.M1(string str, C c)", "c"),
-            // Test0.cs(37,17): warning CA1062: In externally visible method 'void Test.M2(string str, C c)', validate parameter 'c' is non-null before using it. If appropriate, throw an ArgumentNullException when the argument is null or add a Code Contract precondition asserting non-null argument.
-            GetCSharpResultAt(37, 17, "void Test.M2(string str, C c)", "c"));
+            new[]
```

Comment:
:bulb: These explicit arrays are no longer necessary since I defined a `params` form in the verifier helper.